src/README.md

```diff
@@ -40,9 +40,10 @@ heroku git:remote -a "Your app name"
 If you want to create a postgresql, the following command can help you.
 ```
 heroku addons:create heroku-postgresql:hobby-dev
+heroku config:get DATABASE_URL -a "Your app name"
```

Comment:
Actually "heroku addons:create heroku-postgresql:hobby-dev" will set the DATABASE_URL on the heroku config, but you still can get the url by this command.